transform-less.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const readdirp = require('readdirp');
+const childProcess = require("child_process");
+
+const replace = require('replace-in-file');
+
+// Transform less
+(async function() {
+  const files = await readdirp.promise('dist', { fileFilter: '*.less' });
+  files.forEach(file => {
+    const out = file.fullPath.replace('less', 'css');
```

Comment:
It could be worthwile to check for replacing the file extension only as a file named `CarelessComponent.less` would be renamed to `CarecssComponent.less`.